style.css

```
@@ -0,0 +1,1275 @@
+    --text: #1a1a22;
+    --muted: #7a7a8a;
+    --border: #d8d4cc;
+    --shadow: rgba(14,14,18,0.08);
+  }
+
+  * { margin: 0; padding: 0; box-sizing: border-box; }
+
+  body {
+    background: var(--paper);
+    color: var(--text);
+    font-family: 'Lato', sans-serif;
```

Stylelint (17.3.0)

[error] Unexpected quotes around "Lato" (font-family-name-quotes)

(font-family-name-quotes)